website/pages/en/arbitrum/l2-transfer-tools-faq.mdx

```diff
@@ -38,55 +50,141 @@ To transfer your subgraph, you will need to complete the following steps:
+
+### Is there any delegation tax?
+
+No. Received tokens on L2 are delegated to the specified Indexer on behalf of the specified Delegator without charging a delegation tax.
+
+### Will my unrealized rewards be transferred when I transfer my delegation?
+
+​ Yes! The only rewards that can't be transferred are the ones for open allocations, as those won't exist until the Indexer closes the allocations (usually every 28 days). If you've been delegating for a while, this is likely only a small fraction of rewards.
+
+At the smart contract level, unrealized rewards are already part of your delegation balance, so they will be transferred when you transfer your delegation to L2. ​
+
+### Is moving delegations to L2 mandatory? Is there a deadline?
+
+​ Moving delegation to L2 is not mandatory, but indexing rewards are increasing on L2 following the timeline described in [GIP-0052](https://forum.thegraph.com/t/gip-0052-timeline-and-requirements-to-increase-rewards-in-l2/4193). Eventually, if the Council keeps approving the increases, all rewards will be distributed in L2 and there will be no indexing rewards for Indexers and Delegators on L1. ​
+
+### If I am delegating to an Indexer that has already transferred stake to L2, do I stop receiving rewards on L1?
+
+​ Many Indexers are transferring stake gradually so Indexers on L1 will still be earning rewards and fees on L1, which are then shared with Delegators. Once an Indexer has transferred all of their stake, then they will stop operating on L1, so Delegators will not receive any more rewards unless they transfer to L2.
+
+Eventually, if the Council keeps approving the indexing rewards increases in L2, all rewards will be distributed on L2 and there will be no indexing rewards for Indexers and Delegators on L1. ​
+
+### I don't see a button to transfer my delegation. Why is that?
+
+​ Your Indexer has probably not used the L2 transfer tools to transfer stake yet.
+
+If you can contact the Indexer, you can encourage them to use the L2 Transfer Tools so that Delegators can transfer delegations to their L2 Indexer address. ​
+
+### My Indexer is also on Arbitrum, but I don't see a button to transfer the delegation in my profile. Why is that?
+
+​ It is possible that the Indexer has set up operations on L2, but hasn't used the L2 transfer tools to transfer stake. The L1 smart contracts will therefore not know about the Indexer's L2 address. If you can contact the Indexer, you can encourage them to use the transfer tool so that Delegators can transfer delegations to their L2 Indexer address. ​
+
+### Can I transfer my delegation to L2 if I have started the undelegating process and haven't withdrawn it yet?
+
+​ No. If your delegation is thawing, you have to wait the 28 days and withdraw it.
+
+The tokens that are being undelegated are "locked" and therefore cannot be transferred to L2.
```

Comment:
```suggestion
### If I am delegating to an Indexer that has already transferred stake to L2, do I stop receiving rewards on L1?

​Many Indexers are transferring stake gradually so Indexers on L1 will still be earning rewards and fees on L1, which are then shared with Delegators. Once an Indexer has transferred all of their stake, then they will stop operating on L1, so Delegators will not receive any more rewards unless they transfer to L2.

Eventually, if the Council keeps approving the indexing rewards increases in L2, all rewards will be distributed on L2 and there will be no indexing rewards for Indexers and Delegators on L1. ​

### I don't see a button to transfer my delegation. Why is that?

​Your Indexer has probably not used the L2 transfer tools to transfer stake yet.

If you can contact the Indexer, you can encourage them to use the L2 Transfer Tools so that Delegators can transfer delegations to their L2 Indexer address. ​

### My Indexer is also on Arbitrum, but I don't see a button to transfer the delegation in my profile. Why is that?

​It is possible that the Indexer has set up operations on L2, but hasn't used the L2 transfer tools to transfer stake. The L1 smart contracts will therefore not know about the Indexer's L2 address. If you can contact the Indexer, you can encourage them to use the transfer tool so that Delegators can transfer delegations to their L2 Indexer address. ​

### Can I transfer my delegation to L2 if I have started the undelegating process and haven't withdrawn it yet?

​No. If your delegation is thawing, you have to wait the 28 days and withdraw it.

The tokens that are being undelegated are "locked" and therefore cannot be transferred to L2.
```